client/src/components/Styles/Header.scss

```diff
@@ -46,6 +45,22 @@
     }
 }
 
+@media only screen and (min-width: 600px) and (max-width: 900px) {
```

Comment:
Try to made the responsive design by using MUI Breakpoints. https://mui.com/material-ui/customization/breakpoints/